### PR TITLE
CMOR 3.7.3 release notes

### PR DESCRIPTION
**Pull request by mauzey1**

### CODE REVIEW

File: _posts/2023-10-17-cmor3.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags: [news, getting_started]
+---
+
+Changes
+* CMOR now supports Python 3.12.
+
+Bugfixes
+* Fixed error caused when `cmor_write` writes arrays with more than 2<sup>31</sup>-1 elements.
+* Fixed segmentation fault casued when the value for `further_info_url` in the user input is an empty string or only whitespace.
+* Fixed implicit declaration compilation error in cmor.c.
```
Comment (durack1) [inline _posts/2023-10-17-cmor3.md:14]: `cmor.c`?

File: _posts/2023-10-17-cmor3.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Closed Issues
+* [Python 3.12 build](https://github.com/PCMDI/cmor/issues/711){:target="_blank"}
```
Comment (durack1) [inline _posts/2023-10-17-cmor3.md:17]: [#711 Python 3.12 build](https://github.com/PCMDI/cmor/issues/711){:target="_blank"}

File: _posts/2023-10-17-cmor3.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* [CMOR failure when attempting to write large volume of data](https://github.com/PCMDI/cmor/issues/706){:target="_blank"}
```
Comment (durack1) [inline _posts/2023-10-17-cmor3.md:18]: [#706 CMOR failure when attempting to write large volume of data](https://github.com/PCMDI/cmor/issues/706){:target="_blank"}

File: _posts/2023-10-17-cmor3.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* [CMOR segfaults when further_info_url is specified as an empty or pure whitespace string](https://github.com/PCMDI/cmor/issues/700){:target="_blank"}
```
Comment (durack1) [inline _posts/2023-10-17-cmor3.md:19]: [#700 CMOR segfaults when further_info_url is specified as an empty or pure whitespace string](https://github.com/PCMDI/cmor/issues/700){:target="_blank"}

File: _posts/2023-10-17-cmor3.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+* ["call to undeclared function 'calculate_leadtime_coord'" error in recent Xcode/Clang build for OSX](https://github.com/PCMDI/cmor/issues/710){:target="_blank"}
```
Comment (durack1) [inline _posts/2023-10-17-cmor3.md:20]: [#710 "call to undeclared function 'calculate_leadtime_coord'" error in recent Xcode/Clang build for OSX](https://github.com/PCMDI/cmor/issues/710){:target="_blank"}

It'd be useful to order these by issue number for cleanliness

**Review by durack1 — APPROVED** [review]
@mauzey1 looks great - a couple of nits, but otherwise good to merge

File: _posts/2023-10-17-cmor3.md (hunk 1 above)
Comment (mauzey1) [reply _posts/2023-10-17-cmor3.md:14]: It's a fix of a compile error in a C source file.

**Pull request merged**